lab_1_classify_by_unigrams/main.py

```diff
@@ -84,6 +117,23 @@ def compare_profiles(
     In case of corrupt input arguments or lack of keys 'name' and
     'freq' in arguments, None is returned
     """
+    if not (isinstance(unknown_profile, dict) and isinstance(profile_to_compare, dict) and
+            all("name" in a for a in (unknown_profile, profile_to_compare)) and
+            all("freq" in b for b in (unknown_profile, profile_to_compare))):
+        return None
+
+    all_keys = list(set(unknown_profile["freq"]) | set(profile_to_compare["freq"]))
+    corresp_input = [unknown_profile["freq"], profile_to_compare["freq"]]
+    corresp_output = [[], []]
+    for i in all_keys:
+        for j in range(2):
+            if not i in corresp_input[j]:
+                corresp_output[j].append(0)
+            else:
+                corresp_output[j].append(corresp_input[j][i])
+
+    mse = calculate_mse(corresp_output[1], corresp_output[0])
+    return mse
```

Comment:
it is not necessary to save the metric value into a variable

Reply:
Fixed